wire-ios-sync-engine/Source/SessionManager/SessionFactories.swift

```diff
@@ -60,6 +60,7 @@ open class AuthenticatedSessionFactory {
         for account: Account,
         coreDataStack: CoreDataStack,
         configuration: ZMUserSession.Configuration,
+        analyticsSessionConfiguration: AnalyticsSessionConfiguration? = nil,
```

Comment:
I suggest to remove the default value. Default arguments are a source of bugs, in my opinion.